### PR TITLE
fix: remark-lint-no-unused-definitions

### PR DESCRIPTION
**Pull request by nschonni**
Remove unused definitions and correct translation issues

### CODE REVIEW

**Comment by nschonni** [comment]
@nodejs/nodejs-ar need your help with the missing ``[`pipe`][]`` reference from this paragraph https://github.com/nodejs/nodejs.org/blame/18b04185041ccb67e675053195bca13fa6708857/locale/ar/docs/guides/anatomy-of-an-http-transaction.md#L315-L318

File: locale/ar/docs/guides/debugging-getting-started.md [hunk 1]
```diff
@@ -214,5 +214,5 @@ ssh -L 9221:localhost:9229 user@remote.example.com
 
 <!-- refs -->
 
-[بروتوكول المدقق]: https://chromedevtools.github.io/debugger-protocol-viewer/v8/
+[Inspector Protocol]: https://chromedevtools.github.io/debugger-protocol-viewer/v8/
```
Comment (Aissaoui-Ahmed) [inline locale/ar/docs/guides/debugging-getting-started.md:217]: Please change this [line 213](https://github.com/nodejs/nodejs.org/pull/3152/files#diff-346297c51ece9a9262c1aefa70e3ac18R213)

Comment (nschonni) [reply locale/ar/docs/guides/debugging-getting-started.md:217]: Not sure I follow, this is a link used on Line 61

File: locale/ar/docs/guides/anatomy-of-an-http-transaction.md [hunk 2]
```diff
@@ -314,7 +314,7 @@ http.createServer((request, response) => {
 
 رائع! الآن نستقر على تبسيط هذا وتذكر كائنات الطلب `request` هي تدقف قابل للقراءة
 [`ReadableStream`][] و كائنات الجواب `response` هي تدفق قابل للكتابة [`WritableStream`][].
-وهذا يعني أنه يمكننا إستخدام مَجْرىً لتوجيه البيانات من واحدة لأخرى. وهذا تماما مانريده
+وهذا يعني أنه يمكننا إستخدام مَجْرىً لتوجيه البيانات من واحدة لأخرى. وهذا تماما مانريده<!-- fix me -->
```
Comment (Aissaoui-Ahmed) [inline locale/ar/docs/guides/anatomy-of-an-http-transaction.md]:
```suggestion
وهذا يعني أنه يمكننا إستخدام [`pipe`][] لتوجيه البيانات من واحدة لأخرى. وهذا تماما مانريده
```

Comment (nschonni) [reply locale/ar/docs/guides/anatomy-of-an-http-transaction.md]: Thanks! I'll rebase out my junk commit and leave your real fix 👍

File: locale/ar/docs/guides/debugging-getting-started.md (hunk 1 above)
Comment (Aissaoui-Ahmed) [reply locale/ar/docs/guides/debugging-getting-started.md:217]: Oops sorry! 🤦

Comment (nschonni) [reply locale/ar/docs/guides/debugging-getting-started.md:217]: No worries, could have also changed the anchor on 61, but it doesn't look like there is a standard pattern for the translations right now

**Review by Aissaoui-Ahmed — APPROVED** [review]
LGTM 💯

**Pull request merged**